Brokerages/QuantConnect.Brokerages.csproj

```diff
@@ -290,6 +292,9 @@
     <Compile Include="Tradier\UserProfile.cs" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="InteractiveBrokers\IB-symbol-map.json">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
I had some issues with VS 'Newest' some time ago. Not being sure if they fixed it, I am a bit more defensive.